recon_surf/map_surf_label.py

```diff
@@ -155,7 +155,7 @@ def getSurfCorrespondence(
     For each vertex in src_sphere find the closest vertex in trg_sphere.
 
     Spheres are Nx3 arrays of coordinates on the sphere (usually R=100 FS format).
-    Sphere can also be a file name of the sphere.reg files, then we load it.
+    src_sphere can also be a file name of the sphere.reg files, then we load it.
```

Comment:
Due to formatting issues, *_ has been removed.

Reply:
I know, but the meaning was that both src_sphere or tag_sphere can be file names. So both should be mentioned instead of the wildcard.

Reply:
I updated this block and am more specific now (also in the text around)